core/modules/filter/filter.install

```diff
@@ -85,6 +85,20 @@ function filter_update_1001() {
   }
 }
 
+/**
+ * Import the image_library view
+ */
+function filter_update_1002() {
+  $config = config('views.view.image_library');
+  if ($config->isNew()) {
+    $path = backdrop_get_path('module', 'filter') . '/config/';
+    $contents = file_get_contents($path . 'views.view.image_library.json');
```

Comment:
We can't pull in something from the installation defaults into an update hook. In the event that the default changes, it would through off subsequent update hooks. We have to copy the entire view into the update hook. See `file_update_1000()` for an example.

Reply:
OK. I have copied the entire view to an update. Thanks for the example.